packages/lib-classifier/src/plugins/drawingTools/components/Mark/Mark.spec.js

```diff
@@ -197,45 +215,90 @@ describe('Drawing tools > Mark', function () {
   })
 
   describe('useEffect hook', function () {
-    describe('with finished mark and no subTaskVisibility', function () {
-      let markWrapper = (mark) => {
-        return (
+    function markWrapper(mark) {
+      return (
+        <svg>
           <Mark
             label='Point 1'
             mark={mark}
             onDelete={onDelete}
             onFinish={onFinish}
             onSelect={onSelect}
           >
-            <Point mark={point} />
+            <Point mark={mark} />
           </Mark>
-        )
-      }
+        </svg>
+      )
+    }
+
+    describe('with finished mark and no subTaskVisibility', function () {
+      let newMark
+
+      before(function () {
+        window.scrollTo = sinon.stub()
+        newMark = pointTool.createMark()
+        newMark.finish()
+        wrapper = mount(markWrapper(newMark))
+      })
 
-      afterEach(function () {
+      after(function () {
         onFinish.resetHistory()
+        window.scrollTo = undefined
       })
 
-      it('should call onFinish', function () {
-        const newMark = Object.assign({}, point, { finished: true })
+      it('should set subtask visibility', function () {
+        expect(newMark.subTaskVisibility).to.be.true()
+      })
+
+      it('should preserve window scroll position', function () {
+        expect(window.scrollTo).to.have.been.calledOnce()
+      })
+    })
+
+    describe('when the mark is not finished', function () {
+      let newMark
+
+      before(function () {
+        window.scrollTo = sinon.stub()
+        newMark = pointTool.createMark()
         wrapper = mount(markWrapper(newMark))
-        expect(onFinish).to.have.been.calledOnce()
       })
 
-      describe('when the mark is not finished', function () {
-        it('should not call onFinish', function () {
-          const newMark = Object.assign({}, point, { finished: false })
-          wrapper = mount(markWrapper(newMark))
-          expect(onFinish).not.to.have.been.called()
-        })
+      after(function () {
+        onFinish.resetHistory()
+        window.scrollTo = undefined
```

Comment:
Same question and comment about what jsdom expects and restoring that as well as future proofing.